doc/lsst.analysis.ap/tasks/lsst.analysis.ap.PlotImageSubtractionCutoutsTask.rst

```diff
@@ -100,6 +100,23 @@ until then, look at ``data/association-flag-map.yaml`` in `lsst.ap.association`
 
     Cutout with metadata annotations, and all catalog flags set.
 
+**Generating Multi-size Cutouts**
+---------------------------------
+
+Multi-size cutouts can be generated by setting the ``config.sizes`` key in config. The example image shown below contains cutouts of sizes 32x32 and 64x64 pixels.
+
+.. figure:: multisize_cutout_sample-plain.png
+    :name: fig-multisize_cutouts_sample-plain
+    :alt: Multi-size cutouts, with no metadata.
+
+    Multi-size image cutout, with no additional metadata.
+    
+Similarly, the image below shows an example of multi-size cutouts along with all catalog flags set. This can be achieved by setting the ``config.add_metadata`` key to ``True`` in config.
```

Comment:
```suggestion
Similarly, the image below shows an example of multi-size cutouts along with all catalog flags set.
This can be achieved by setting the ``config.add_metadata`` key to ``True``.
```